CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This is the log of notable changes to EAS CLI and related packages.
 ### 🐛 Bug fixes
 
 - Compute runtime version policies. ([#783](https://github.com/expo/eas-cli/pull/783) by [@jkhales](https://github.com/jkhales))
+- Fix accidentally requiring runtime version for all builds. ([#785](https://github.com/expo/eas-cli/pull/785) by [@jkhales](https://github.com/jkhales))
```

Comment:
```suggestion
- Compute runtime version policies. ([#783](https://github.com/expo/eas-cli/pull/783), [#785](https://github.com/expo/eas-cli/pull/785) by [@jkhales](https://github.com/jkhales))
```
No need to add separate entry because we haven't released it yet.